cypress/integration/functional/wallet/settings/privacyLock.spec.ts

```diff
@@ -0,0 +1,22 @@
+function enablePrivacyLock (): void {
+  localStorage.setItem('Development.Local.PRIVACY_LOCK.enrolled', 'TRUE')
+  cy.wait(500)
+  expect(localStorage.getItem('Development.Local.PRIVACY_LOCK.enrolled')).equal('TRUE')
+}
+
+context('Wallet - Privacy Lock', () => {
```

Comment:
Quite minimal tests here, with artificially "enabling" privacy lock in web env which has no biometric hardware.